src/System Application/App/Edit in Excel/src/Filters/EditInExcelFiltersImpl.Codeunit.al

```diff
@@ -120,11 +122,12 @@ codeunit 1491 "Edit in Excel Filters Impl."
                             ChildNodesJsonToken.AsObject(),
                             ODataJsonPayload,
                             FieldFilterGroupingOperator,
-                            "Edit in Excel Filter Collection Type".FromInteger(ExcelFilterNodeType.AsInteger()));
+                            "Edit in Excel Filter Collection Type".FromInteger(ExcelFilterNodeType.AsInteger()),
```

Comment:
Why the line break here?

Reply:
@JesperSchulz Just a continuation of what we already had, do you want everything on one line or?

Reply:
I think this might be an issue with how Github wraps the line, and it looks like there is a line break in the middle of the enum name.
![image](https://github.com/microsoft/BCApps/assets/55883409/2fe1e353-f924-4566-a030-bd89cb76bb1a)

But you can see that the line number does not increase, and if you go to the three dots on the top right of the file and click View File, indentation/newlines actually look fine IMHO :)